.snippets/code/develop/toolkit/api-libraries/polkadot-js-sdk/read-constants.js

```diff
@@ -0,0 +1,3 @@
+// Get the minimum balance required for a new account
+const minBalance = api.consts.balances.existentialDeposit.toNumber();
+console.log('Minimum Balance:', minBalance);
```

Comment:
I'd remove this line since the most important part of the whole snippet:

`api.consts.balances.existentialDeposit.toNumber();`

```suggestion
```